code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1478,7 +1478,7 @@
 
 /datum/reagent/medicine/changelinghaste/on_mob_metabolize(mob/living/L)
 	..()
-	L.add_movespeed_modifier(type, update=TRUE, priority=100, multiplicative_slowdown=-2, blacklisted_movetypes=(FLYING|FLOATING))
+	L.add_movespeed_modifier(type, update=TRUE, priority=100, multiplicative_slowdown=-0.75, blacklisted_movetypes=(FLYING|FLOATING))
```

Comment:
I think mustle strain should remain high, acting as something you can literally only use for running away because of the stun you get at the end of it (and the fact the speed is too high to actually fight)

Reply:
I guess...